styles/templates/_VOWLDatatypeObjectProperty.scss

```diff
@@ -0,0 +1,132 @@
+.ontodia-VOWLDatatypeObjectProperty_template {
+  margin: auto;
+  max-height: 53px;
+  max-width: 170px;
+  border-style: solid;
+  border-width: 2px;
+
+  &__picture {
+    margin: 5px 0px;
+    border: 1px solid;
+  }
+  &__picture-image {
+    width: 100%;
+  }
+
+  &__name {
+    display: block;
+    width: 100%;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    font-size: 19px;
+    font-weight: 100;
+  }
+}
+
+.ontodia-VOWLDatatypeObjectProperty_template_body {
```

Comment:
Use SASS nested selectors for styles here, e.g.:
```
.ontodia-vowl-property-template {
  &__body {
    ...
  }

  &__body-type-line {
    ...
  }
}
```

Reply:
done